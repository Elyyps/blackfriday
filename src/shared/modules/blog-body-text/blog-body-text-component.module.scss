.blog-body-text {
  @media (max-width: $breakpoint-xsmall-max) {
    padding: 0 !important;
  }
  &__container {
    box-sizing: border-box;
    width: 100%;
    min-width: 100%;
    display: inline-flex;
    flex-direction: row;
    @media (max-width: $breakpoint-xsmall-max) {
      flex-direction: column;
      padding: 0;
      padding-left: 0;
      padding-right: 0;
    }
  }
  &__content {
    flex: 1 1 72%;
    box-sizing: border-box;
    padding: 16px;
    padding-right: 8%;

    @media (max-width: $breakpoint-xsmall-max) {
      width: 100%;
      padding-left: 16px;
      padding-right: 16px;
    }
    &__footer {
      border-top: 1px solid #e0e0e0;
      padding: 16px 0;
    }
    &__header {
      color: #777777;
      font-size: 14px;
      margin-bottom: 24px;
      display: inline-flex;
      align-items: center;
      line-height: 1.29;
      svg {
        margin-right: 8px;

        path {
          fill: #777;
        }
      }
    }
  }
  &__sidebar {
    flex: 1 1 27%;
    @media (max-width: $breakpoint-xsmall-max) {
      width: 100%;
    }
  }
  @media (max-width: $breakpoint-xsmall-max) {
    .blog-body-text {
      flex-direction: column;
      &__content {
        width: 100%;
      }
      &__sidebar {
        width: 100%;
      }
    }
  }
}
