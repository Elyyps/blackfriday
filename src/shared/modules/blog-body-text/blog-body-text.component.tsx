
import * as React from "react";
import styles from "./blog-body-text-component.module.scss";

export interface IBlogBodyTextComponentProps {

}

const BlogBodyTextComponent = (props: IBlogBodyTextComponentProps) => {
  return ( <div className={styles["blog-body-text"]}>

  </div>)
};

export { BlogBodyTextComponent };
  