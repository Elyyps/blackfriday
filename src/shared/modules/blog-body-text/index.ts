export * from './blog-body-text.component'