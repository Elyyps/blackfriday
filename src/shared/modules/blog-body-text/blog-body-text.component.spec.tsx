
import React from "react";
import { shallow } from "enzyme";
import { BlogBodyTextComponent } from "./blog-body-text.component";

describe("[BlogBodyText]", () => {
  it("should render component without crashing", () => {
    const renderedComponent = shallow(
      <BlogBodyTextComponent />
    );
    expect(renderedComponent).toMatchSnapshot();
  });
});

  