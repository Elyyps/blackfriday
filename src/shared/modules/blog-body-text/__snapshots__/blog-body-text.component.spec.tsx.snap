// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[BlogBodyText] should render component without crashing 1`] = `
<div
  className="uk-container blog-body-text"
>
  <div
    className="blog-body-text__container"
  >
    <div
      className="blog-body-text__content"
    >
      <div
        className="blog-body-text__content__header"
      >
        <IconComponent
          icon="clock.svg"
          size="14"
        />
         
        2 minuten leestijd
      </div>
      <BodyTextComponent
        bodyTextModule={
          Object {
            "bottomPadding": Object {
              "desktopSpacing": 72,
              "mobileSpacing": 32,
            },
            "containerAlignment": "center",
            "containerSize": "full",
            "html": "<h2>
        Hogere kortingen, meer producten en meer uiteenlopende productcategorieÃ«n. Dit gaat er gebeuren met Black
        Friday.
      </h2>
      <h2>1. Wees er snel bij, winkels beginnen eerder met Black Friday</h2>
      <p>
        Vorig jaar zagen we deze trend al opkomen. Winkels willen altijd graag de eerste zijn met hun deal, daarom
        beginnen een aantal winkels niet op Black Friday, maar vaak een paar dagen eerder. Het is daarom belangrijk dat
        je niet alleen op 23 november 2018 <a href=\\"#\\">Black Friday Nederland</a> goed in de gaten houdt, maar ook
        in de week voor de kortingsdag van het jaar de website al iedere dag even checkt. Je weet nooit wanneer de deals
        online komen. Toch zien we wel dat de allerbeste deals pas echt vanaf Black Friday ingaan, maar wees er dan snel
        bij! Want voordat je het weet is jouw favoriete product uitverkocht.
      </p>
      <h2>2. Groots uitpakken met Telecom deals</h2>
      <p>
        Telecom deals zijn niet meer weg te denken van Black Friday. Wij verwachten dat er dit jaar nog meer Telecom
        deals online komen dan vorig jaar. Ben je dus op zoek naar een <a href=\\"#\\">mobiel</a>? Wacht dan nog even
        tot Black Friday want we verwachten dat providers als <a href=\\"#\\">Vodafone</a>, <a href=\\"#\\">T-Mobile</a>
        , <a href=\\"#\\">BEN</a> en <a href=\\"#\\">Tele2</a> weer groots uit gaan pakken met de smartphone deals van
        2018. De Samsung Galaxy of de Apple Iphone zijn smartphones waar je altijd wel een fikse korting op kunt vinden.
        Kun je niet wachten? Bekijk dan alvast de <a href=\\"#\\">Samsung Galaxy deals</a> van dit moment.
      </p>
      <h2>3. 45% meer deelnemende winkels in 2019</h2>
      <img src=\\"https://viper-development-images.s3-eu-west-1.amazonaws.com/blackFriday/blog/3.png\\" width=\\"279px\\" height=\\"146px\\" className=\\"align-right\\" alt=\\"image description\\" />
      <p>
        Naar verwachting doen er in 2018 bijna de helft meer winkels mee aan <a href=\\"#\\">Black Friday 2018</a>.
        Daardoor zal het aantal deals ook bijna met de helft stijgen. Het is dus belangrijk dat je jouw favoriete winkel
        goed in de gaten houdt, want voordat je het weet wordt je ondergesneeuwd door de kortingen en kun je jouw deal
        niet meer vinden.
      </p>
      <h2>4. Meer lokale scherpe aanbiedingen</h2>
      <p>
        Behalve grote namen als <a href=\\"#\\">adidas</a>, <a href=\\"#\\">MediaMarkt</a>, <a href=\\"#\\">Ziggo</a> en
        <a href=\\"#\\">Bol.com</a> is het ook slim om dit jaar de kleine lokale winkels goed in de gaten te houden.
        Naar verwachting zullen de kleine winkels en boetiekjes in jouw dorp of stad ook meedoen aan Black Friday. Black
        Friday Nederland richt zich echter alleen op de landelijke bekende winkels, dus de aanbiedingen van dat kleine
        winkeltje in jouw dorp zul je zelf in de gaten moeten houden. Dit kan vaak al door de winkel te volgen op Social
        Media en even te vragen of ze dit jaar meedoen aan Black Friday.
      </p>
      <h2>5. Meer producten in reis-, interieur- en fashionbranche</h2>
      <p>
        Dit jaar verwachten wij dat vooral de reisbranche uit gaat pakken. Deze speler was er vorig jaar ook bij met
        onder andere merken als: <a href=\\"#\\">Ryanair</a>, Hotels.com, <a href=\\"#\\">Expedia</a> en Hilton. De
        verwachting is vooral dat er kortingen komen op vliegtickets en hotels met Black Friday 2018. Houd onze
        reispagina goed in de gaten voor een recent overzicht van alle <a href=\\"#\\">reisaanbiedingen</a> met Black
        Friday.
      </p>",
            "id": "1",
            "name": "BlogBodyText",
            "topPadding": Object {
              "desktopSpacing": 72,
              "mobileSpacing": 32,
            },
          }
        }
      />
      <div
        className="blog-body-text__content__footer"
      >
        <ShareSocialDropdownComponent
          buttonTitle="Delen"
          shareSocial={
            Object {
              "email": Object {
                "nameSocialShare": "Email",
                "subject": "BlackFriday",
                "title": "Email",
                "url": "https://www.webbio.nl",
              },
              "facebook": Object {
                "nameSocialShare": "Facebook",
                "title": "BlackFriday",
                "url": "https://www.webbio.nl",
              },
              "twitter": Object {
                "hashtags": Array [
                  "BlackFriday",
                  "Netherlands",
                  "Sale",
                  "Shops",
                ],
                "nameSocialShare": "Twitter",
                "title": "BlackFriday",
                "url": "https://www.webbio.nl",
                "via": "BlackFriday",
              },
              "whatsapp": Object {
                "nameSocialShare": "WhatsApp",
                "separator": " - ",
                "title": "BlackFriday",
                "url": "https://www.webbio.nl",
              },
            }
          }
        />
      </div>
    </div>
    <div
      className="blog-body-text__sidebar"
    >
      <SidebarComponent
        banner={
          Object {
            "buttonLink": "/deals-overview",
            "buttonTitle": "Meer deals",
            "image": Object {
              "alt": "Ad picture",
              "src": "https://viper-development-images.s3-eu-west-1.amazonaws.com/blackFriday/ad.png",
            },
            "imageLink": "https://coolblue.nl",
            "label": "22 t/m 26 nov.",
            "logo": Object {
              "alt": "CoolBlue logo",
              "src": "cool-blue.svg",
            },
            "moreInfoLink": "https://coolblue.nl",
            "showAlternativeBanner": false,
            "text": "Een selectie van bizarre aanbiedingen.",
            "title": "Black Friday bij Coolblue",
          }
        }
        featuredItemsGroups={
          Array [
            Object {
              "featuredItems": Array [
                Object {
                  "image": "imac.png",
                  "link": Object {
                    "title": "Apple iPhones",
                    "url": "/stores-single",
                  },
                },
                Object {
                  "image": "iphone.png",
                  "link": Object {
                    "title": "Apple MacBooks",
                    "url": "/stores-single",
                  },
                },
                Object {
                  "image": "loader.png",
                  "link": Object {
                    "title": "Apple iMacs",
                    "url": "/stores-single",
                  },
                },
                Object {
                  "icon": "party.svg",
                  "link": Object {
                    "title": "Naar alle deals!",
                    "url": "/stores-single",
                  },
                },
              ],
              "title": "Een greep uit onze deals",
            },
            Object {
              "featuredItems": Array [
                Object {
                  "image": "cool-blue.svg",
                  "link": Object {
                    "title": "Coolblue",
                    "url": "/stores-single",
                  },
                },
                Object {
                  "image": "cool-blue.svg",
                  "link": Object {
                    "title": "Alternate",
                    "url": "/stores-single",
                  },
                },
                Object {
                  "image": "cool-blue.svg",
                  "link": Object {
                    "title": "Beddenreus",
                    "url": "/stores-single",
                  },
                },
                Object {
                  "icon": "party.svg",
                  "link": Object {
                    "title": "Naar alle winkels!",
                    "url": "/stores-overview",
                  },
                },
              ],
              "title": "Black Friday Winkels",
            },
          ]
        }
      />
    </div>
  </div>
</div>
`;
